### PR TITLE
Add workflows for MBR Windows Server builds

### PR DESCRIPTION
**Pull request by jjerger**
Adds MBR workflows to add to Concourse later

### CODE REVIEW

File: daisy_workflows/build-publish/windows/windows-server-2012-r2-dc-bios.publish.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-server-2012-r2-dc-bios",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{if eq .environment "prod" -}}
```
Comment (bkatyl) [inline daisy_workflows/build-publish/windows/windows-server-2012-r2-dc-bios.publish.json:14]: For all of these publish files, I think they should only be able to publish to `bct-prod-images`

Comment (jjerger) [reply daisy_workflows/build-publish/windows/windows-server-2012-r2-dc-bios.publish.json]: Follow up: Is the byol license the right one here? I figure we don't want to be using the billed ones for testing?

Comment (jjerger) [reply daisy_workflows/build-publish/windows/windows-server-2012-r2-dc-bios.publish.json]: Discussed offline to use payg instead of byol

**Review by bkatyl — APPROVED** [review]

**Pull request merged**